deployment/README.md

```diff
@@ -28,6 +28,23 @@ You will be prompted to enter your AWS credentials, along with a default region.
 
 ## Terraform
 
+### Setup
+
+Make sure there is a `terraform/terraform.tfvars` file in the project config bucket on S3 for the
+environment you're deploying (i.e. `s3://{environment}-pfb-config-us-east-1/terraform/terraform.tfvars`).
+If you're creating a new environment, make a new file via copy/paste/modify or by defining all the
+variables specified in [terraform/variables.tf].  If you're updating an environment and need to
```

Comment:
Should there be a link here? Or should these be backticks instead of square brackets?

Reply:
Oops, yeah, meant to link to the file.